src/appleseed.studio/utility/miscellaneous.cpp

```diff
@@ -170,6 +170,17 @@ QString get_project_files_filter(const int filters)
     return filter_list.join(";;");
 }
 
+QString get_resumable_render_files_filter()
```

Comment:
Unless this is needed in several places, move it to where it's needed. You can even get rid rid of the function and simply pass the filter as a string.